darts/models/forecasting/block_rnn_model.py

```diff
@@ -464,6 +512,7 @@ def _create_model(self, train_sample: Tuple[torch.Tensor]) -> torch.nn.Module:
             num_layers=self.n_rnn_layers,
             num_layers_out_fc=hidden_fc_sizes,
             dropout=self.dropout,
+            activation=self.activation,
```

Comment:
Adding `activation` will break loading the weights from earlier python versions.

You would have to add the method `_check_ckpt_parameters` to handle this case.
You can look at the TiDEModel as reference. Here would be the method for the BlockRNNModel:

```
    def _check_ckpt_parameters(self, tfm_save):
        # new parameters were added that will break loading weights
        new_params = ["activation"]
        for param in new_params:
            if param not in tfm_save.model_params:
                tfm_save.model_params[param] = "ReLU"
        super()._check_ckpt_parameters(tfm_save)
```

Having done this, only the case when a user trained a model with multiple fc layers will fail (which is okay, since no activation there is bad practice in any case).